ibis/backends/dask/execution/util.py

```diff
@@ -142,3 +228,16 @@ def compute_sorted_frame(
     result = df.assign(**{sort_col_name: temporary_column})
     result = result.set_index(sort_col_name).reset_index(drop=True)
     return result
+
+
+def assert_identical_grouping_keys(*args):
+    indicies = [arg.index for arg in args]
+    # Depending on whether groupby was called like groupby("col") or
+    # groupby(["cold"]) index will be a string or a list
+    if isinstance(indicies[0], list):
+        indicies = [index[0] for index in indicies]
```

Comment:
Can index contain multiple elements? groupby(["cold", "warm"])?

Reply:
Errr - yes actually. Sorry! Not sure why I thought they didn't? Will fix the logic here.

Reply:
Sounds good. Can you add a test for this?